pkg/relayer/proxy/error_reply.go

```diff
@@ -13,20 +14,22 @@ import (
 // TODO_TECHDEBT: This method should be aware of the nature of the error to use the appropriate JSONRPC
 // Code, Message and Data. Possibly by augmenting the passed in error with the adequate information.
 func (jsrv *jsonRPCServer) replyWithError(writer http.ResponseWriter, err error) {
-	relayResponse := &types.RelayResponse{
-		Payload: &types.RelayResponse_JsonRpcPayload{
-			JsonRpcPayload: &types.JSONRPCResponsePayload{
-				Id:      0,
-				Jsonrpc: "2.0",
-				Error: &types.JSONRPCResponseError{
-					// Using conventional error code indicating internal server error.
-					Code:    -32000,
-					Message: err.Error(),
-					Data:    nil,
-				},
-			},
+	response := map[string]interface{}{
```

Comment:
Interesting. 

Can you update the comment (and potentially doc) that even though we are making `payload` general and transparent, we are still in the process of determining if we should have protocol-aware errors. 

I want to think about this more, but it's not a bad design.

Reply:
Do you suggest just replying directly with the payload response bytes we receive no matter what? That would make the errors specific to the service which is probably ideal for the end user